README.md

```diff
@@ -65,12 +65,17 @@ $ ./local/check-dependencies.sh
 ```
 
 Most dependencies are easily installed using a package manager, such as `brew`.
-However, there are installation scripts for some dependencies, such as:
+However, there are installation scripts for all dependencies, such as:
 
 ```
 $ ./local/install-dependency-3proxy.sh
 ```
 
+You can automatically install all missing dependencies by passing in the install flag
+```
+$ ./local/check-dependencies.sh install
```

Comment:
I think we should make this `--install` to make it more clear that this is a toggle/flag.